roles/ood_uab_ui/templates/dashboard/env_local.j2

```diff
@@ -0,0 +1,8 @@
+RAILS_RELATIVE_URL_ROOT="/pun/sys/{{ app_name }}"
+OOD_DASHBOARD_TITLE="UAB OnDemand <sup>beta</sup>"
+OOD_DASHBOARD_DOCS_URL="https://docs.uabgrid.uab.edu"
+BOOTSTRAP_NAVBAR_INVERSE_BG="{{ navbar_inverse_bg }}"
+OOD_PORTAL="ondemand"
+
+MOTD_PATH="/etc/motd"
+MOTD_FORMAT="txt"
```

Comment:
Let's change this to markdown. Since it will consume motd off login003 we can use that as a testing space for motd improvements and markdown will enhance our formatting options.

I'm assuming our stock motd content will render just the same whether we use txt or markdown.